modules/programs/firefox/mkFirefoxModule.nix

```diff
@@ -236,7 +236,7 @@ in {
       default = false;
       example = true;
       description = ''
-        Whether to enable ${name}.${
+        Whether to enable ${cfg.name}.${
```

Comment:
Unfortunately this is not OK since it makes the manual output depend on your configuration, causing unnecessary rebuilds and makes the pre-built manual inconsistent with the manual an individual user has.

So, regrettably this needs to be reverted. If the output needs to be revised then it will have to be done without any reference to `config`.

Reply:
Oh right sorry about that! I'll submit a fix that just references the attribute.